paket.dependencies

```diff
@@ -18,6 +18,8 @@ nuget NuGet.CommandLine
 nuget Microsoft.CodeAnalysis.Analyzers 1.1
 nuget Microsoft.CodeAnalysis 2.0
 
+nuget Microsoft.Composition >= 1.0.31
```

Comment:
Why do we need the higher version? Which version of Visual Studio is required to satisfy this?

Reply:
The older version only shipped netfx/pcl builds, and we relied on the package target fallback.

With the new SDK for netcoreapp it includes a few default fallbacks which conflict with the one we defined, so I had to remove them.

The new version supports netstandard too. I believe this is only used by unit tests, in any event.

Reply:
Seems reasonable.

>  I believe this is only used by unit tests, in any event.

I thought the unit tests would use the version in the 'Test' group.